<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TWinEditAbonent</class>
 <widget class="QDialog" name="TWinEditAbonent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактировать параметры абонента</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IconsForMenu.qrc">
    <normaloff>:/1/Picture/Pic1.ico</normaloff>:/1/Picture/Pic1.ico</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>107</y>
     <width>261</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="OKButton">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>38</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="icon">
       <iconset resource="IconsForMenu.qrc">
        <normaloff>:/1/Picture/OKGreen.png</normaloff>:/1/Picture/OKGreen.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelButton">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>38</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
      <property name="icon">
       <iconset resource="IconsForMenu.qrc">
        <normaloff>:/1/Picture/OKRed.png</normaloff>:/1/Picture/OKRed.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>41</y>
     <width>325</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IP-адрес абонента</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EditAddress">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>9</y>
     <width>325</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Имя абонента</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EditName">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>73</y>
     <width>325</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Номер TCP-порта</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EditPort">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="IconsForMenu.qrc"/>
 </resources>
 <connections/>
</ui>
